up-l1/README.adoc

```diff
@@ -122,9 +122,21 @@ On the other hand, unsuccessful completion of this method does not necessarily m
 
 NOTE: The above strategy for retrying failed attempts to send a message results in https://www.cloudcomputingpatterns.org/at_least_once_delivery/[at-least-once delivery]. Recipient(s) of these messages should therefore be https://www.cloudcomputingpatterns.org/idempotent_processor/[Idempotent Processors].
 
-UTransport implementations
 
+[.specitem,oft-sid="des~utransport-send-preserve-data~1",oft-needs="impl,utest"]
+--
 * *MUST* preserve all of the message's meta data and payload during transmission
+--
+
+[.specitem,oft-sid="req~utransport-send-error-invalid-parameter~1",oft-needs="impl,utest"]
+--
+* *MUST* fail with a `UCode.INVALID_ARGUMENT` if the passed UMessage failed validation
+--
+
+[.specitem,oft-sid="req~utransport-send-error-permission-denied~1",oft-needs="impl,utest"]
```

Comment:
FMPOV it is not obvious, how this should be implemented. Thus, this requirement will probably need some implementation specific design first.

```suggestion
[.specitem,oft-sid="req~utransport-send-error-permission-denied~1",oft-needs="dsn"]
```

Reply:
We define what the UMessage validators are suppose to do so why do we need more implementation specific design first?

Reply:
This is about the spec item for the _permission denied_ case, not the _invalid message_ ...